debian/postrm

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+
+case "$1" in
+    purge|remove)
+        if [ -d "/var/lib/ubuntu-advantage" ]; then
+            rm -rf /var/lib/ubuntu-advantage
+        fi
```

Comment:
I think we don't want this on remove, just purge

Reply:
ah yes I'll fix